documentation/book/proc-deploying-kafka-bridge-quickstart.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-quickstart.adoc
+
+[id='proc-deploying-kafka-bridge-quickstart-{context}']
+= Deploying the Kafka Bridge to your Kubernetes cluster
+
+{ProductName} includes a YAML template that specifies the configuration of the {ProductName} Kafka Bridge. Make some minimal changes to this template and then deploy an instance of the Kafka Bridge to your Kubernetes cluster.
```

Comment:
I would talk about "example" rather than "template". The "template" word always let me think about OpenShift templates which are something different.

Reply:
OK, changed to "example".